api/libproject/devstart.rb

```diff
@@ -882,6 +882,44 @@ def set_authority(cmd_name, *args)
   :fn => lambda { |*args| set_authority("set-authority", *args) }
 })
 
+def delete_clusters(cmd_name, *args)
+  ensure_docker cmd_name, args
+  op = WbOptionsParser.new(cmd_name, args)
+  op.opts.dry_run = true
+  op.add_option(
+      "--min-age-days [DAYS]",
+      lambda {|opts, v| opts.min_age_days = v},
+      "Optional minimum age filter in days for clusters to delete, e.g. 21")
+  op.add_option(
+      "--ids [CLUSTER_ID1,...]",
```

Comment:
If we are going to provide an ID flag to this, does it make sense to also provide a way to get a list of cluster IDs? Or is this more for the use case of someone going "My cluster doesn't work! Please delete/restart it!"

If that is the case, does it make sense to have this be a single cluster ID, rather than a list?

Reply:
I suspect it will be both. Note: we'll also be giving them a button to restart themselves, if they get stuck, so I don't imagine us using this flag too much in general.

Supporting the ability to list clusters is a good idea, I will add that and reply here again when done. I think it makes sense to keep the flag plural though given that this tool is pretty slow to start up right now.

Reply:
Add a `list-clusters` command as well.